### DISCUSSION TITLE
Remove 74 Bracket

### DISCUSSION THREAD

**Discussion by Shardaz** [chromiecraft/chromiecraft#6407] [Suggestions]
I know it is some time ahead, but given the planning time this might need, I'd like to start this discussion early. 

I would like to suggest removing the 74 bracket, going directly from 70 to 79.

I think the 64 bracket was the worst bracket we experienced on Chromiecraft so far.

- the outland open world was never meant to be for lvl 64. (cant really explain, but beeing in the lower classic brackets felt WAY better) The main stuff you could so was farming stuff you were basically too low level. Like Scryers/Aldor stuff, high level crafting mats etc. This was just a painful necessity to stay caught up with other players. Casters having a huge disatvatage because of hit mechanics.

- Not enough relevant content. You still went into the old raids/dungeons because you were honored with Thrallmar/Expedition on day 3 and then nothing else to do. Just leaving old contend somewhat iffy caused by different stat scaling, mana cost of next level spells, beeing 64 in general

- people were looking for glitches to get into high level zones anyways, forcing cc to add teleport dragons

- a lot of people lost interest in the game. 64 bracket felt the most "dead"

Please reconsider making the same error for wotlk again. I think most players would prefer staying longer in swp bracket instead of moving to a extremely limited and unbalanced wotlk situation.

**Comment by heyitsbench** [comment]
Forget not that (if I'm not mistaken) the reasoning for doing an intermediary bracket is to help find issues with the lower level dungeons. If the bracket jumped straight up to 69/79, the dungeons would only see enough runs to level out of them and no more testing would be done, which is against the point of the bracket system.

**Comment by balleny** [comment]
tbh, i though the 64 bracket was one of the better ones.

raids became accessible for people that werent able to clear yet, people got easier ways to catch-up with gear, people could "challenge" themselves and farm level 70 stuff / items.

level 64 felt a lot more active than zg / aq40 bracket
